lib/dfe_sign_in_api.rb

```diff
@@ -13,7 +13,7 @@ def users(page: 1)
       raise ExternalServerError if response.code.eql?(500)
       raise ForbiddenRequestError if response.code.eql?(403)
 
-      JSON.parse(response) if response.code.eql?(200)
+      JSON.parse(response&.body) if response.code.eql?(200)
```

Comment:
Do you have a spec covering this change?

Reply:
Its difficult to test with `webmock` as the tests are passing with both `response` and `response.body`

Reply:
I think we found this because we got an `HTTParty::Response` Error when the job ran in the local env.
Maybe we should have a test around not getting this particular error? Not sure.